src/seedu/addressbook/data/person/Person.java

```diff
@@ -25,6 +27,7 @@ public Person(Name name, Phone phone, Email email, Address address, UniqueTagLis
         this.email = email;
         this.address = address;
         this.tags = new UniqueTagList(tags); // protect internal tags from changes in the arg list
+        this.sequenceNumber = ++nextSequenceNumber;
```

Comment:
Good. `sequenceNumber` should start with 1. It's a good idea to add a comment here to explain the pre-increment.